src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -159,12 +159,34 @@ public static Tag parseTag(String tag) throws ParseException {
     public static LocalDateTime parseAppointmentDateTime(String dateTime) throws ParseException {
         requireNonNull(dateTime);
         String trimmedDateTime = dateTime.trim();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+        String[] dateTimeInfo = trimmedDateTime.split(" ");
+
+        if (dateTimeInfo.length != 2) {
```

Comment:
Possible to distinguish whether date or time is missing or maybe both?

Reply:
I think this check is sufficient as we are reminding the user to make sure both date and time are present.